### PR TITLE
[UI] User Interface for forms added

### PR DESCRIPTION
**Pull request by imjalpreet**
@nisargjhaveri and @gauravmittal1995 Please take a look at it and suggest any changes, if required.

### CODE REVIEW

**Comment by gauravmittal1995** [comment]
M sure its fine, but i will check it later. If anyone else checks it before me and approves it, i can merge it, otherwise will merge it after i check it.

File: app/views/missing/report.php [hunk 1]
```diff
@@ -1,51 +1,256 @@
...
+                                        <label class="sr-only" for="lname">Last name: </label>
+                                        <input type="text" class="form-control" placeholder="Enter your Last Name" id="lname" name="person[lname]" required /><br>
+                                        <label class="sr-only" for="gender">Gender: </label>
+                                        <label class="sr-only" for="dob">Date of birth: </label>
+                                        <input type="date" placeholder="Enter your DOB" class="form-control" id="dob" type="date" name="person[dob]" /><br>
+                                        <select id="gender" name="person[gender]" required>
+                                            <option value="" disabled selected>Select gender</option>
+                                            <option value="F">Female</option>
+                                            <option value="M">Male</option>
+                                            <option value="O">Other</option>
```
Comment (ParthKolekar) [inline app/views/missing/report.php]: :grin:

**Comment by nisargjhaveri** [comment]
Some suggestions/questions/problems
- [ ] Can't we keep all the groups open by default in both the pages? It is too many clicks to fill and submit the form. Also difficult to find what I'm looking for.
- [ ] The styling of Gender drop-down?
- [ ] The "Report missing" button is too large (it might be just personal preference). 
- [ ] Color of "Report missing" button. It is not easy to separate it out from section heads.
- [ ] Why is everything on left, can't we put it in center. Also, when we open the sidebar, it shrinks, even if it has more space on right.

**Comment by nisargjhaveri** [comment]
Btw, works perfectly. Good job! :+1:

**Comment by imjalpreet** [comment]
@nisargjhaveri I will update the design asap :+1:

File: app/views/missing/report.php [hunk 2]
```diff
@@ -1,51 +1,256 @@
...
+                        <?php
+                        if ($reported !== null) {
+                            if ($reported == true) {
+                                echo "Reported successfully";
+                            }
+                            else {
+                                echo "Submission failed";
+                            }
+                            echo "<br><br>";
+                        }
```
Comment (nisargjhaveri) [inline app/views/missing/report.php:123]: Also, this might be shown in a little nicer way.

Comment (gauravmittal1995) [inline app/views/missing/report.php:123]: use alert-success and alert-danger.

Comment (imjalpreet) [inline app/views/missing/report.php:123]: Yeah, that's what I am using now!

**Comment by imjalpreet** [comment]
I have changed the UI. Just have a look at it

**Comment by nisargjhaveri** [comment]
Looks good. :+1:

**Pull request merged**